### DISCUSSION TITLE
Why is removed page still accessible

### DISCUSSION THREAD

**Discussion by chilikasha** [vercel/community#317] [Help] [answered]
Hi,

I have a site made with docusaurus and hosted/deployed on vercel. It used to have a blog, which was later removed completely. But still `/blog/archive` page can be opened (it has top navigation bar and no content).
Why doesn't it return 404?

**Comment by Pieparker** [comment, marked as answer]
Sounds like the page is still being built by the framework. You'll want to [consult the Docusaurus guide/notes](https://docusaurus.io/docs/creating-pages) on how to add or remove pages and avoid them from being built!
